jujugui/static/gui/src/app/state/state.js

```diff
@@ -82,6 +95,70 @@ const State = class State {
   }
 
   /**
+    Check the supplied block of path to see if this is a user store path or not.
+    @return {Boolean}
+  */
+  _isUserStorePath(block) {
+    const seriesList = this.seriesList;
+    if (block.length === 3 && block[1] === 'bundle') {
+      return false;
+    }
+    if (block.length === 2 && !seriesList.includes(block[1])) {
+      return Number.isNaN(parseInt(block[1], 10));
+    }
+    return !Number.isNaN(parseInt(block[2], 10)) ||
+      seriesList.includes(block[2]);
+  }
+
+  /**
+    The current window location. If set has been called on this property then
+    it will return the externally set option. Typically this option will be used
+    for testing.
+    @type {Object}
+  */
+  get location() {
+    return this._location || window.location;
+  }
+
+  set location(location) {
```

Comment:
I thought so too, but apparently you're only to document one of the set with jsdoc.